ipam/allocator.go

```diff
@@ -70,6 +70,8 @@ func NewAllocator(lcDs, glDs datastore.DataStore) (*Allocator, error) {
 		}
 	}
 
+	a.checkConsistency(localAddressSpace)
```

Comment:
It needs more thinking. In a  mutlihost scenario, the consistency check operation has to be done only once.
If every host does it, then a a daemon booted later on a second host may destroy addresses allocated in the meanwhile by the daemon on the first host. Also at the moment I do not know how to handle the case where one host runs 1.9.1 and another host runs 1.10.